src/com/untamedears/realisticbiomes/GrowthConfig.java

```diff
@@ -261,103 +238,65 @@ public boolean isPersistent() {
 	}
 	
 	// given a block (a location), find the growth rate using these rules
+	
 	public double getRate(Block block) {
 		if (type == Type.FISHING_DROP) {
 			return getBaseRate(block);
 		}
-		// rate = baseRate * sunlightLevel * biome * (1.0 + soilBonus)
-		double rate = baseRate;
-		// if persistent, the growth rate is measured in growth/second
+		double rate= getBaseRate(block);
+		if (rate==0D) {
+			return rate;
+		}
 		if (isPersistent) {
 			rate = 1.0 / (persistentRate * SEC_PER_HOUR);
+			Double biomeMultiplier = biomeMultipliers.get(block.getBiome());
+			rate *= biomeMultiplier.floatValue();

```

Comment:
It seems like the biome rate would only be applied if persistent!
And `biomeMultiplier` can be null, in which case the function used to return zero, because it is a biome that shouldn't exist on civcraft *I think*.

I think it should return zero in both cases, null biome or if floatValue() is zero, since you have done away with greenhouseIgnoreBiome, yay!